test/e2e/vsphere_volume_diskformat.go

```diff
@@ -43,16 +43,16 @@ import (
 	2. Create PVC which uses the StorageClass created in step 1.
 	3. Wait for PV to be provisioned.
 	4. Wait for PVC's status to become Bound
-	5. Create POD using PVC on specific node.
+	5. Create pod using PVC on specific node.
 	6. Wait for Disk to be attached to the node.
 	7. Get node VM's devices and find PV's Volume Disk.
 	8. Get Backing Info of the Volume Disk and obtain EagerlyScrub and ThinProvisioned
 	9. Based on the value of EagerlyScrub and ThinProvisioned, verify diskformat is correct.
-	10. Delete POD and Wait for Volume Disk to be detached from the Node.
+	10. Delete pod and Wait for Volume Disk to be detached from the Node.
 	11. Delete PVC, PV and Storage Class
 */
 
-var _ = framework.KubeDescribe("Volume Disk Format", func() {
+var _ = framework.KubeDescribe("Volume Disk Format [Feature:Volume] [vsphere]", func() {
```

Comment:
I thought it would be easier to filter vsphere related tests with adding [vsphere] tag.
but if this is not the convention, let’s remove this.